Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_044.NpmCredentialsValidator.cs

```diff
@@ -101,7 +101,9 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NotFound ||
                     responseWithNoCredentials.StatusCode == HttpStatusCode.NonAuthoritativeInformation)
                 {
-                    return ValidationState.NoMatch;
+                    message = $"The Uri {uri} is generic, no way to validate teh credential with for account {id}";
```

Comment:
>message [](http://example.com/codeflow?start=20&length=7)

can you show an example of entire message of unknown results?

i remember this message will be concat with another message so in some case it starts with a space and lowercase.

Reply:
npm-cli-login -u userNameHere -p P@s$w0rdHere S -e emailHere@microsoft.com -r https://registry.npmjs.org